eventhub/Module.md

```diff
@@ -58,33 +25,44 @@ event hubs. It includes operations to revoke publisher, get revoked publishers,
 
 * An Event Hub https://docs.microsoft.com/en-us/azure/event-hubs/event-hubs-create#create-an-event-hub 
 
+### Obtaining Tokens
 * Connection String of the Event Hub Namespace
-We need management credentials to communicate with the Event Hubs. These credentials are available in the connection 
-string of the Event Hub namespace. Obtain the connection string for the Event Hubs namespace by following the 
-instructions given below.
-https://docs.microsoft.com/en-us/azure/event-hubs/event-hubs-get-connection-string#get-connection-string-from-the-portal
+
+    We need management credentials to communicate with the Event Hubs. These credentials are available in the connection 
```

Comment:
Shall we break them into steps. 

Management credentials are required to communicate with the Event Hub.
1.1. Follow steps in <link> and obtain the connection string. 
1.2. Get the management credentials from the connections string by...

Reply:
Fixed in df7c75b . Thank you.